test/suites/integration/tags_test.go

```diff
@@ -86,7 +86,7 @@ var _ = Describe("Tags", func() {
 	})
 
 	Context("Tagging Controller", func() {
-		It("should tag with karpenter.sh/nodeclaim and Name tag", func() {
+		It("should tag with karpenter.sh/nodeclaim, karpenter.k8s.aws/ec2nodeclass, and Name tag", func() {
```

Comment:
Do you mind expanding this tests to include all the tags we expect on the instance? The only tags that are left are the `karpenter.sh/nodepool` and `kubernetes.io/cluster/$ClusterName`

Reply:
Also for this PR, I was thinking we could have another test to validate the NodeClaim has the label. Can most likely go here https://github.com/aws/karpenter-provider-aws/blob/main/test/suites/nodeclaim/nodeclaim_test.go

Reply:
Added.